homework/english/kataya_pechinskaya/lecture1/task2.md

```diff
@@ -0,0 +1,12 @@
+#Homework 1 task 2
+#Найдимо было найти ошибки и исправьте их.
+1. We not going to school today. - We aren't going to school today.
+2. What you doing after school? - What are you doing after school? 
+3. At the moment Peter is work in Russia. - At the moment Peter is working in Russia.
+4. Does he got a new car? - Does he get a new car? -
+5. He never wear a hat. - He never wears a hat.
+6. He don’t like black coffee. - He doesn’t like black coffee.
+7. We are have a good time. - We have a good time.
```

Comment:
this variant can be used, but 'We are having a good time' better
because we don't always have a good time